boots.py

```diff
@@ -683,7 +684,7 @@ def remotelock(configuration):
                 '--include', 'Linux', 'CPython', version, 'x86_64',
                 '--include', 'macOS', 'CPython', version, 'x86_64',
                 '--archive-file', archive_path,
-                '--artifact-paths', artifact_paths,
+                '--artifact-paths', f"'{artifact_paths}'",
```

Comment:
I know I suggested this but this is Python 2/3 compatible code so we can't f-string (3.6+).

```suggestion
                '--artifact-paths', "'{}'".format(artifact_paths),
```